tasks/deploy/mainnet-config.ts

```diff
@@ -153,7 +156,8 @@ const addresses: ExtSystemConfig = {
         ],
         assetsIn: ["0xA13a9247ea42D743238089903570127DdA72fE44", "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0"],
     },
-    lzEndpoint: "0x66A71Dcef29A0fFBDBE3c6a460a3B5BC225Cd675",
+    canonicalChainId: 101,
```

Comment:
Similar comment regarding the l1LzEndpoint in that this can just be called `lzChainId` as it's already in the context of the mainnet config.